### ISSUE TITLE
2.3.5 not compatible with Xcode 6

### ISSUE THREAD

**Issue by jtreanor** [intercom/intercom-ios#98]
When building an app with 2.3.5 in Xcode 6, you may see errors like these:

```
duplicate symbol _llvm.cmdline in: 
path_to_Intercom/Intercom.framework/Intercom(AFHTTPRequestOperation.o) 
path_to_Intercom/Intercom/Intercom.framework/Intercom(SRWebSocket.o)

duplicate symbol _llvm.embedded.module in: 
path_to_Intercom/Intercom/Intercom.framework/Intercom(AFHTTPRequestOperation.o) 
path_to_Intercom/Intercom.framework/Intercom(SRWebSocket.o) 
...
```

As pointed out by @mfkp, this may be related to http://www.openradar.me/21884601. We are looking into fixes for this at the moment.

In the meantime you can either use 2.3.4 of Intercom or update to Xcode 7.

**Comment by jtreanor** [comment]
This has been fixed in 2.3.6. Intercom should now work fine with older versions of Xcode.

**Comment by mfkp** [comment]
Awesome, thanks for the quick fix @jtreanor, works great :+1:
